### PR TITLE
Project_documentation_Kalman Temperature Alert System

### PR DESCRIPTION
**Pull request by Fote20003**
### Pull Request Overview

This pull request adds/changes/fixes...

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs...

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Your Name <Your Email@example.com>

### CODE REVIEW

**Review by irina-b-dev — COMMENTED** [review]
Please put the index.md in a folder named : maria-irina.fotescu_teo-antonio.economu

**Review by irina-b-dev — COMMENTED** [review]
Make a single branch and pull request for both authors.

File: docs/project/2024/Project_documentation/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Kalman Temperature Alert System
+
```
Comment (alexandruradovici) [inline docs/project/2024/Project_documentation/index.md:2]: Please add a one line description of the project here.

File: docs/project/2024/Project_documentation/index.md
Comment (alexandruradovici) [inline docs/project/2024/Project_documentation/index.md:1]: Please rename the folder to your curs.uob.ro username.

File: docs/project/2024/Project_documentation/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+The format is
+
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+
+-->
+
+| Device                                                                                                  | Usage               | Price                                                                                                                                                                                                                                                                           |
+| ------------------------------------------------------------------------------------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (alexandruradovici) [inline docs/project/2024/Project_documentation/index.md:57]:
```suggestion
|-|-|-|
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Closing as duplicate of #242.

**Pull request closed without merging**